modules/create_timepoint/test/create_timepointTest.php

```diff
@@ -162,16 +158,13 @@ private function _createTimepoint($canID, $subproject, $visitlabel)
      */
     function testCreateTimepointErrorEmptySubproject()
     {
-        $this->markTestSkipped(
-            'Skipping tests until create timepoint works well'
-        );
         $this->safeGet(
             $this->url . "/create_timepoint/?candID=900000&identifier=900000"
         );
         $this->webDriver->findElement(WebDriverBy::Name("fire_away"))->click();
         $bodyText = $this->webDriver->getPageSource();
-        $this->assertContains(
-            "A visit label is required for creating a timepoint.",
+        $this->assertNotContains(
```

Comment:
I don't see why this change is required. Is the error message not being displayed or has the wording changed?